roboteam_ai/src/coach/CoachHeuristics.h

```diff
@@ -15,32 +15,33 @@ namespace ai {
 namespace coach {
 
 class CoachHeuristics {
-    private:
-        using WorldData = world::WorldData;
-        using Ball = world::Ball;
-        using Robot = world::Robot;
-        using BallPtr = world::World::BallPtr;
-        using RobotPtr = world::World::RobotPtr;
-
-        static const double MAX_DISTANCE_FROM_BALL;
-        static const double CLOSE_TO_GOAL_WEIGHT;
-        static const double SHOT_AT_GOAL_WEIGHT;
-        static const double PASS_LINE_WEIGHT;
-        static const double DISTANCE_TO_OPPONENTS_WEIGHT;
-        static const double DISTANCE_FROM_CORNER_WEIGHT;
-    public:
-        static double calculateCloseToGoalScore(const Vector2 &position);
-        static double calculateShotAtGoalScore(const Vector2 &position, WorldData world);
-        static double calculatePassLineScore(const Vector2 &position, WorldData world);
-        static double calculateDistanceToOpponentsScore(const Vector2 &position);
-        static double calculateDistanceFromCornerScore(const Vector2 &position, roboteam_msgs::GeometryFieldSize field);
-        static double calculateDistanceFromBallScore(const Vector2 &position, roboteam_msgs::GeometryFieldSize &field,
-                BallPtr &ball);
-        static double calculatePositionScore(const Vector2 &position);
+private:
+    using WorldData = world::WorldData;
+    using Ball = world::Ball;
+    using Robot = world::Robot;
+    using BallPtr = world::World::BallPtr;
+    using RobotPtr = world::World::RobotPtr;
+
+    static const double MAX_DISTANCE_FROM_BALL;
+    static const double CLOSE_TO_GOAL_WEIGHT;
+    static const double SHOT_AT_GOAL_WEIGHT;
+    static const double PASS_LINE_WEIGHT;
+    static const double DISTANCE_TO_OPPONENTS_WEIGHT;
+    static const double DISTANCE_FROM_CORNER_WEIGHT;
+public:
+    static double calculateCloseToGoalScore(const Vector2& position);
+    static double calculateShotAtGoalScore(const Vector2& position, WorldData world);
+    static double calculatePassLineScore(const Vector2& position, WorldData world);
+    static double calculateDistanceToOpponentsScore(const Vector2 &position, const WorldData world);
+    static double calculateDistanceFromCornerScore(const Vector2& position, const roboteam_msgs::GeometryFieldSize& field);
```

Comment:
preferably for references &  : Vector2 &position or Vector2 & position
and for pointers *                  : Vector2* position or Vector2 * position